lib/linux_admin/network_interface.rb

```diff
@@ -51,8 +51,10 @@ def reload
 
       @network_conf[:mac] = parse_ip_output(ip_output, %r{link/ether}, 1)
 
-      ip_route_res = Common.run!(Common.cmd("ip"), :params => ["route"])
-      @network_conf[:gateway] = parse_ip_output(ip_route_res.output, /^default/, 2) if ip_route_res.success?
+      [4, 6].each do |version|
```

Comment:
This all might be cleaner if we move the call to `run!` and the rescue into a separate method (`ip_route(version)` maybe?).
Then the assignment will just be two lines like:
```ruby
@network_conf[:gateway]  = parse_ip_output(ip_route(4), /^default/, 2)
```

That should also make this all easier to unit test.